src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -237,23 +237,29 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
                 this.fcClusterConfig = FrameworkControllerClusterConfigFactory.generateFrameworkControllerClusterConfig(frameworkcontrollerClusterJsonObject);
-                if(this.fcClusterConfig.storageType === 'azureStorage') {
-                    let azureFrameworkControllerClusterConfig = <FrameworkControllerClusterConfigAzure>this.fcClusterConfig;
-                    this.azureStorageAccountName = azureFrameworkControllerClusterConfig.azureStorage.accountName;
-                    this.azureStorageShare = azureFrameworkControllerClusterConfig.azureStorage.azureShare;
-                    await this.createAzureStorage(
-                        azureFrameworkControllerClusterConfig.keyVault.vaultName,
-                        azureFrameworkControllerClusterConfig.keyVault.name,
-                        azureFrameworkControllerClusterConfig.azureStorage.accountName,
-                        azureFrameworkControllerClusterConfig.azureStorage.azureShare
-                    );
-                } else if(this.fcClusterConfig.storageType === 'nfs') {
-                    let nfsFrameworkControllerClusterConfig = <FrameworkControllerClusterConfigNFS>this.fcClusterConfig;
-                    await this.createNFSStorage(
-                        nfsFrameworkControllerClusterConfig.nfs.server,
-                        nfsFrameworkControllerClusterConfig.nfs.path
-                    );
-                } 
+                try{
+                    if(this.fcClusterConfig.storageType === 'azureStorage') {
+                        let azureFrameworkControllerClusterConfig = <FrameworkControllerClusterConfigAzure>this.fcClusterConfig;
+                        this.azureStorageAccountName = azureFrameworkControllerClusterConfig.azureStorage.accountName;
+                        this.azureStorageShare = azureFrameworkControllerClusterConfig.azureStorage.azureShare;
+                        await this.createAzureStorage(
+                            azureFrameworkControllerClusterConfig.keyVault.vaultName,
+                            azureFrameworkControllerClusterConfig.keyVault.name,
+                            azureFrameworkControllerClusterConfig.azureStorage.accountName,
+                            azureFrameworkControllerClusterConfig.azureStorage.azureShare
+                        );
+                    } else if(this.fcClusterConfig.storageType === 'nfs') {
+                        let nfsFrameworkControllerClusterConfig = <FrameworkControllerClusterConfigNFS>this.fcClusterConfig;
+                        await this.createNFSStorage(
+                            nfsFrameworkControllerClusterConfig.nfs.server,
+                            nfsFrameworkControllerClusterConfig.nfs.path
+                        );
+                    } 
+                } catch(error) {
+                    this.log.error(error);
+                    return Promise.reject(new Error(error));
+                }
```

Comment:
I assume the problem is caused by that the type of error is string?  if yes, the problem has been address by this PR, https://github.com/microsoft/nni/pull/1064 , it is not necessary to handle this issue case by case.